README.md

```diff
@@ -460,6 +460,86 @@ crud.upsert_object('customers',
 ...
 ```
 
+### Batch upsert
+
+Right now CRUD cannot provide batch upsert with full consistency.
+CRUD offers batch upsert with partial consistency. That means
+that full consistency can be provided only on single replicaset
+using `box` transactions.
+
+```lua
+-- Batch upsert tuples
+local result, err = crud.batch_upsert(space_name, tuples, operations, opts)
```

Comment:
Well... shouldn't it be an array with (tuple, operations) records rather than same operations for each tuple?

To be honest, I think both situations may be useful in production and maybe both of them should be supported. We should ask ticket authors about what they expected to do with upsert because I don't have any experience of using upserts in real code.

Reply:
Updated, now we support `upset_many` with different operations for each tuple.